app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>All Ingredients</h1>
+<% @ingredients.each do |ingredient| %>
+<p><%= ingredient.name%> </P>
```

Comment:
This looks good just be mindful about how it is asking to be displayed on the page. 
```
User Story 1 - Ingredients Index

As a visitor,
When I visit '/ingredients'
I see a list of all the ingredients including their name and cost
(e.g. "Ground Beef: 2", "Salt: 4")
```
So in your view you could do:
```ruby
<% @ingredients.each do |ingredient| %>
  <p><%= ingredient.name %>: <%= ingredient.cost %></p>
<% end %>
```